tests/test_avro_schema.py

```diff
@@ -0,0 +1,168 @@
+"""
+    These are duplicates of other test_schema.py tests, but do not make use of the registry client fixture
+    and are here for debugging and speed, and as an initial sanity check
+"""
+from avro.schema import Parse
+from karapace.avro_compatibility import ReaderWriterCompatibilityChecker, SchemaCompatibilityType
+
+import json
+import pytest
+
+
+def test_simple_schema_promotion():
+    reader = Parse(json.dumps({"name": "foo", "type": "record", "fields": [{"type": "int", "name": "f1"}]}))
+    writer = Parse(
+        json.dumps({
+            "name": "foo",
+            "type": "record",
+            "fields": [{
+                "type": "int",
+                "name": "f1"
+            }, {
+                "type": "string",
+                "name": "f2"
+            }]
+        })
+    )
+    res = ReaderWriterCompatibilityChecker().get_compatibility(reader, writer)
+    assert res.compatibility is SchemaCompatibilityType.compatible, res
+    res = ReaderWriterCompatibilityChecker().get_compatibility(writer, reader)
+    assert res.compatibility is SchemaCompatibilityType.incompatible, res
+
+    writer = Parse(
+        json.dumps({
+            "type": "record",
+            "name": "CarrierAssignment",
+            "namespace": "lastmile.data.model",
```

Comment:
Hello,
I'm part of the team that discover this bug and provides this schema, can you _anonymize_ it ? Like removing _lastmile_

Best regards

Reply:
done

Reply:
thanks :+1